blueprint/cli/src/bin/db.rs

```diff
@@ -152,7 +152,8 @@ async fn create(config: &DatabaseConfig) -> Result<String, anyhow::Error> {
 
 async fn migrate(ui: &mut UI<'_>, config: &DatabaseConfig) -> Result<i32, anyhow::Error> {
     let db_config = get_db_config(config);
-    let migrator = Migrator::new(Path::new("db/migrations"))
+    let migrations_path = format!("{}/../db/migrations", env!("CARGO_MANIFEST_DIR"));
```

Comment:
I was surprised that CARGO_MANIFEST_DIR points to /cli instead of the root project folder

Reply:
Wow, nice work! I thought it'd be a bit more of a challenge ;)

Reply:
thanks! :D